### PR TITLE
p9: registry.Get fix message type bug

### PR DESCRIPTION
**Pull request by sevki**
Signed-off-by: sevki <s@sevki.org>

### CODE REVIEW

File: p9/messages.go [hunk 1]
```diff
@@ -2027,6 +2027,9 @@ type registry struct {
 // This takes, and ignores, a message tag so that it may be used directly as a
 // lookuptagAndType function for recv (by design).
 func (r *registry) get(_ tag, t msgType) (message, error) {
+	if t >= math.MaxUint8 {
```
Comment (hugelgupf) [inline p9/messages.go:2030]: t is a uint8. wouldn't it be easier to do `factories [math.MaxUint8+1]msgFactory`? (I think either is valid)

Comment (sevki) [reply p9/messages.go:2030]: the exact same comment on the gvisor side :smile:  https://github.com/google/gvisor/pull/3317#discussion_r458362078

I'll update both :)

**Comment by sevki** [comment]
@hugelgupf PTAL

File: p9/messages.go (hunk 1 above)
Comment (hugelgupf) [reply p9/messages.go]: i learned a lot from them :)

**Pull request merged**